test/diff-details/expected-stdout

```diff
@@ -5,7 +5,6 @@ CH changed
 Diff:
 CH B bar
   "description":
-    -%c
```

Comment:
Prazdne hodnoty uz nezobrazujeme, bolo to trocha metuce `-(medzera)` a dalej nic,
vyzeralo to, ako keby tam bol predtym prazdny riadok, ale v skutocnosti je `remote` description uplne prazdna, a `local` description obsahuje `new description`.

Reply:
`%c` tam bolo predtym preto, lebo IDE vzdy pri ulozeni odstranilo medzeru z konca riadka.